interfaces/builtin/all_test.go

```diff
@@ -332,6 +332,20 @@ apps:
         plugs: [iface]
 `
 
+const testInvalidSlotInterfaceYaml = `
+name: testsnap
+slots:
+ foo:
+  interface: iface
+`
+
+const testInvalidPlugInterfaceYaml = `
+name: testsnap
+plugs:
+ bar:
+  interface: iface
```

Comment:
Can you please include an app and a hook here as well. Make them use the plug.

Nitpick: please rename `bar to just `iface`.